### ISSUE TITLE
Support multiple KC instances in a single namespace

### ISSUE THREAD

**Issue by vmuzikar** [keycloak/keycloak#10562]
* Double check it's possible to deploy multiple CRs (of various types) in a single namespace representing multiple Keycloak instances.
* Add tests for deploying two KC instances in a single namespace together with all CR types.

**Comment by jsorah** [comment]
I ran into this while doing some testing with multiple CRs defined in a single namespace.  I was able to limit to some specific pods using the statefulset, but that seems less than ideal.

This spurred a couple of thoughts I wanted to capture:

- Should the operator have its own "label" namespace such as `operator.keycloak.org`  where it would be used like `operator.keycloak.org/some-label: value` ?
- It seems like the operator could populate the label it is using to identify the pods for services with a value specific to the CR, maybe even just the CR name itself.
- It would be nice if additional labels could be specified in the CR and applied to all of the resources the operator creates.
  - This probably should be its own feature request of the operator.
